tests/virt_autotest/libvirt_extend_storage_lvm.pm

```diff
@@ -102,7 +104,13 @@ sub get_hard_disk_size {
 # Wipe Hard Disk Clean via dd
 sub wipe_hard_disk {
     my $hard_disk_name = shift;
+    my $timeout = 180;
+    record_info("Wipe Hard Drive $hard_disk_name forcibly");
     assert_script_run("dd if=/dev/zero of=$hard_disk_name count=1M", timeout => 1500, fail_message => "Failed to wipe hard disk clean on $hard_disk_name");
+    record_info("Remove GPT signature $hard_disk_name forcibly");
+    script_run("sgdisk --zap-all $hard_disk_name", timeout => $timeout);
```

Comment:
Based on code logic, it seems this cmd is necessary and shall be successful . Then assert_script_run is better?

Reply:
This is not needed any more. I just realize the default behavior of script_run is also die after timeout.